ehsm-service/napiparams.js

```diff
@@ -0,0 +1,119 @@
+const keyspec= {
+  type : 'const',
+  arr: ['EHM_AES_GCM_128','EHM_RSA_3072'],
+  errortext: "keyspec is must 'EHM_AES_GCM_128' or 'EHM_RSA_3072'",
```

Comment:
same here, the name not match with ehsm_provider.h

Reply:
It will be fix.